rollup/Makefile

```diff
@@ -7,9 +7,22 @@ mock_abi:
 	cd .. && solc --evm-version cancun --bin --abi --optimize --overwrite -o ./build/bin ./rollup/mock_bridge/MockBridge.sol
 	cd .. && go run github.com/scroll-tech/go-ethereum/cmd/abigen --bin=./build/bin/MockBridge.bin --abi=./build/bin/MockBridge.abi --pkg=mock_bridge --out=./rollup/mock_bridge/MockBridge.go
 
-rollup_bins: ## Builds the Rollup bins.
+libzstd:
+	sudo mkdir -p /scroll/lib/
+	sudo wget -O /scroll/lib/libzktrie.so https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+	sudo wget -O /scroll/lib/libscroll_zstd.so https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libscroll_zstd.so
```

Comment:
we can make the <scroll_version> a variable, just like https://github.com/scroll-tech/scroll/blob/612e66ebc201cb504ee0f518ed91405d88eec944/Makefile#L3-L4

Reply:
make sense.

Reply:
will change it later.

Reply:
changed in [b215558](https://github.com/scroll-tech/scroll/pull/1298/commits/b215558a423efda04c2edeadbd7e518f4fe8190c).